src/views/TabView/TabBarBottom.js

```diff
@@ -71,10 +71,18 @@ export default class TabBarBottom extends PureComponent<DefaultProps, Props, voi
       inputRange,
       outputRange,
     });
+    const label = this.props.getLabelText(scene);
+    if (typeof label === 'string') {
+      return (
+        <Animated.Text style={[styles.label, { color }]}>
+          {label}
+        </Animated.Text>
+      );
+    }
     return (
```

Comment:
Isn't the default value (the `routeName`) passed down a string though, meaning by default it'll lose all of the styling? My thought was:

```
label: null // Render the routeName
label: 'Custom Label' // Render a custom string but with default styling
label: <Text>Custom Component</Text> // Render it inside Animated.View
```

With your proposed changes, the first two will lose all default styling. Unless I'm being an idiot :D